tools/docs/config.ts

```diff
@@ -233,10 +237,65 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
         `\n${option.description}\n\n` +
         genTable(Object.entries(el), option.type, option.default);
 
+      if (el.advancedUse) {
```

Comment:
How do we intend to test if the code works? In production, via code review? I don't see any tests, but maybe it's hard to write test for code like this?

Reply:
I already ran the `build:docs` command and compared the previous docs with the new ones.
As I see there are no tests for this right now.